src/cpu/dyn_cache.h

```diff
@@ -741,7 +745,12 @@ static inline void dyn_mem_execute()
 		pthread_jit_write_protect_np(true);
 #elif defined(HAVE_MPROTECT)
 	if (mprotect(cache_code_start_ptr, cache_code_size, PROT_EXEC | PROT_READ) == -1)
-		E_Exit("mprotect failed in dyn_mem_execute");		
+		E_Exit("mprotect failed in dyn_mem_execute");
+#elif defined(WIN32)
+	DWORD old = 0;
```

Comment:
old is a bit generic (or not that self-documenting); maybe something like `previous_protection`, `prior_protect_flags`, .. ?